K2Field.K2NE.ServiceBroker/ServiceObjectBase.cs

```diff
@@ -56,7 +56,7 @@ public string LDAPPaths
         {
             get
             {
-                return this.ServiceBroker.Service.ServiceConfiguration[Constants.ConfigurationProperties.LDAPPaths].ToString();
+                return this.ServiceBroker.Service.ServiceConfiguration[Constants.ConfigurationProperties.LDAPPaths].ToStringOrEmpty();
```

Comment:
Are we sure that returning EMPTY is better here? Why not simply throw an exception so a proper error message is provided to the end user?